dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
 mock
 nose
 pylint
+ruamel.yaml
```

Comment:
If regular `yaml` isn't being really developed anymore, lets just kill it and move to using `ruamel.yaml`